src/activities/ActivityUsageEntity.js

```diff
@@ -261,7 +261,7 @@ export class ActivityUsageEntity extends Entity {
 	 * @returns {bool} Whether or not the to show the due date
 	 */
 	showDueDate() {
-		return !this.isReusableActivity() && this._entity.hasSubEntityByClass(Classes.dates.dueDate);
+		return !this.isReusableActivity() && this.dueDate() !== undefined;
```

Comment:
just confirming, `dueDate == null` is a valid due date value right?

Also since dueDate can return a string, would an empty string be valid? `''` Or maybe that's impossible?

Reply:
Assignment with no due date

```
{
      "type": "text",
      "name": "dueDate",
      "value": ""
  },```